src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/Tokenization.cs

```diff
@@ -188,10 +200,26 @@ private static bool CanHideLeftHandSideOfDottedName(NameInfo nameInfo, TexlBindi
         /// <param name="tokens">Collection in which tokens created from th given nodes would be stored.</param>
         /// <param name="binding">Binding which would be used to tokenize the operation and determine the type of each token.</param>
         /// <param name="compilerGeneratedNodes">Collection that contains ids of compiler generate nodes.</param>
-        private static void ExtractTokensFromStringInterpolationNodesList(ICollection<ITokenTextSpan> tokens, TexlBinding binding, ICollection<int> compilerGeneratedNodes)
+        /// <param name="tokenTypesToSkip">Options: Collection of token types to potentially skip in the final result. Tokens with these token types may or may not be skipped depending on their types.</param>
+        private static void ExtractTokensFromStringInterpolationNodesList(ICollection<ITokenTextSpan> tokens, TexlBinding binding, ICollection<int> compilerGeneratedNodes, IReadOnlyCollection<TokenType> tokenTypesToSkip = null)
```

Comment:
> IReadOnlyCollection<TokenType> tokenTypesToSkip = null) [](http://example.com/codeflow?start=171&length=56)

I am assuming, all the signature change doesn't break PA.

Reply:
Nope it doesn't. Plus i modified an internal method which is only used language server in a private function. this is not anywhere near to "public" and PA doesn't use only language server does